pkg/tests/api/alerting/api_prometheus_test.go

```diff
@@ -717,6 +720,7 @@ func TestPrometheusRulesPermissions(t *testing.T) {
 	}
 
 	// remove permissions from folder2
+	// TODO this too
```

Comment:
Note: I forgot to address this - we should change these folder ACL updates to the corresponding RBAC updates.